app/src/main/java/com/keylesspalace/tusky/AccountActivity.kt

```diff
@@ -141,6 +142,7 @@ class AccountActivity : BottomSheetActivity(), ActionButtonActivity, HasAndroidI
         animateEmojis = sharedPrefs.getBoolean(PrefKeys.ANIMATE_CUSTOM_EMOJIS, false)
         hideFab = sharedPrefs.getBoolean("fabHide", false)
 
+        handleWindowInsets()
```

Comment:
is it enough to call it in `onCreate()` only? if yes, I have no other notes. Otherwise I would put it to something like `onResume()`